src/components/ScmRepositoryName/ScmRepositoryName.tsx

```diff
@@ -0,0 +1,18 @@
+import { Link } from 'react-router-dom';
+
+import { parseScmRepositoryTitle } from 'utils/utils';
+
+interface IScmRepositoryNameProps {
+  url: string;
```

Comment:
this can be also used for external links, see here: https://github.com/project-ncl/pnc-web-ui-react/pull/172/files#diff-8479f044baf9e5f796f6d9bd53bfdb647cb94eb750de3ce7b645218f5cda8b45R99

Reply:
ok, will fix it accordingly